extensions/amp-story/1.0/animation.js

```diff
@@ -436,8 +485,10 @@ export class AnimationRunner {
 
   /** @private */
   notifyFinish_() {
-    if (this.target_.id) {
-      this.sequence_.notifyFinish(this.target_.id);
+    // TODO(alanorozco): This should work with <amp-story-animation> by
+    // exposing it as a sequencing id.
```

Comment:
Can you update this comment to say why it doesn't, or what's needed to make it work, or create a ticket?